Firmata.cpp

```diff
@@ -93,12 +93,14 @@ void FirmataClass::printVersion(void)
 
 void FirmataClass::blinkVersion(void)
 {
+#if defined(VERSION_BLINK_PIN)
   // flash the pin with the protocol version
   pinMode(VERSION_BLINK_PIN, OUTPUT);
-  strobeBlinkPin(FIRMATA_MAJOR_VERSION, 40, 210);
+  strobeBlinkPin(VERSION_BLINK_PIN, FIRMATA_MAJOR_VERSION, 40, 210);
   delay(250);
-  strobeBlinkPin(FIRMATA_MINOR_VERSION, 40, 210);
+  strobeBlinkPin(VERSION_BLINK_PIN, FIRMATA_MINOR_VERSION, 40, 210);
```

Comment:
Ideally but it has been defined in Boards.h for years. I'm not sure if anyone is using it in their sketches - probably not because it is only used in Firmata.cpp, but nothing prevents anyone from using it in a sketch as well. This is a issue I've been having lately. Each new Arduino board has included a duplicate define somewhere in it's code base that has broken something in Firmata so prefixing is necessary... just trying to figure out a way to do it without breaking existing sketches.